src/config/subgraph.ts

```diff
@@ -8,7 +8,7 @@ const SUBGRAPH_URLS = {
     referrals: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/gmx-arbitrum-referrals/api",
     nissohVault: "https://api.thegraph.com/subgraphs/name/nissoh/gmx-vault",
     syntheticsStats: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/synthetics-arbitrum-stats/api",
-    subsquid: "https://gmx.squids.live/gmx-synthetics-arbitrum/graphql",
+    subsquid: "https://gmx-test.squids.live/hubert-gmx-arbitrum/v/v1/graphql",
```

Comment:
Just a reminder to not forget to change to the production version before merging